Engine/game/savegame.h

```diff
@@ -0,0 +1,202 @@
+//=============================================================================
+//
+// Adventure Game Studio (AGS)
+//
+// Copyright (C) 1999-2011 Chris Jones and 2011-20xx others
+// The full list of copyright holders can be found in the Copyright.txt
+// file, which is part of this source code distribution.
+//
+// The AGS source code is provided under the Artistic License 2.0.
+// A copy of this license can be found in the file License.txt and at
+// http://www.opensource.org/licenses/artistic-license-2.0.php
+//
+//=============================================================================
+
+#ifndef __AGS_EE_GAME__SAVEGAME_H
+#define __AGS_EE_GAME__SAVEGAME_H
+
+#if defined (_MSC_VER)
+#include <memory>
+#elif defined (__GNUC__)
+#include <tr1/memory>
```

Comment:
no, it is std::tr1 namespace, which we are using for a about year in the new version. Should compile with latest gcc 4.